tests/functional/ddb/test_select.py

```diff
@@ -0,0 +1,385 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import json
+
+import ruamel.yaml as yaml
+
+from awscli.testutils import BaseAWSCommandParamsTest
+
+
+class BaseSelectTest(BaseAWSCommandParamsTest):
+    def assert_yaml_response_equal(self, response, expected):
+        with self.assertRaises(ValueError):
+            json.loads(response)
+        loaded = yaml.safe_load(response)
+        self.assertEqual(loaded, expected)
+
+
+class TestSelect(BaseSelectTest):
+    def setUp(self):
+        super(TestSelect, self).setUp()
+        self.parsed_response = {
+            "Count": 1,
+            "Items": [{"foo": {"S": "spam"}}],
+            "ScannedCount": 1,
+            "ConsumedCapacity": None,
```

Comment:
Any idea on when I run a select command , I get a `ConsumedCapacity: null` in the output printed to me? Running with `--debug`, it does not show it being returned in the wire response either. Historically, CLI commands do not print the parameter if it is not available.

Reply:
This will happen in `dynamodb` commands too if they're paginated. I'll have to remove that after the fact.